student.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parents_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly fix the initialize method to request `age`, `name` and `parents_permission`.

- [ ] Also the `#super` method arguments are missing the correct order.

  ```
  def initialize(age, classroom, parents_permission: true, name: 'Unknown')
    super(age, parents_permission: parents_permission, name: name)
    @classroom = classroom
  end
  ```